webview/src/plots/components/ribbon/styles.module.scss

```diff
@@ -7,47 +7,45 @@
   top: 3px;
 }
 
-.columnsTable {
-  td {
-    padding: 4px;
-
-    &:first-child {
-      text-align: left;
-    }
-
-    &:nth-child(2) {
-      padding-right: 0;
-    }
-
-    &.depsKey {
-      color: $deps-color;
-    }
-
-    &.metricsKey {
-      color: $metrics-color;
-    }
-
-    &.paramsKey {
-      color: $params-color;
-    }
-
-    svg {
-      display: block;
-      min-width: 100%;
-      min-height: 100%;
-    }
-
-    .copyButton {
-      opacity: 0;
-      display: inline;
-      position: static;
-      margin-top: 2px;
-      font-size: 0.8125rem;
-    }
-
-    &:hover .copyButton {
-      opacity: 1;
-    }
+.tooltipColumn {
```

Comment:
Almost all changes in the pr were done with the `--fix` flag and I made three manual changes:

* `no-descending-specificity` issue in `/ribbon` 
* `no-descending-specificity` issue in `/plots` 
* `at-extend-no-missing-placeholder` in `/table`